### ISSUE TITLE
Unable to find the game "MsPacmen"

### ISSUE THREAD

**Issue by ewanlee** [mail-ecnu/Text-Gym-Agents#10]
(Possible) reproduce:

Create a Python script in the root directory:

```python
import gym
from atariari.benchmark.wrapper import AtariARIWrapper
env = AtariARIWrapper(gym.make('MsPacmanNoFrameskip-v4'))
obs = env.reset()
obs, reward, trunc, done, info = env.step(1)
```

<img width="1120" alt="image" src="https://github.com/mail-ecnu/Text-Gym-Agents/assets/11542337/cdeba946-5708-4551-9e29-d23d0c789c33">

**Comment by Jarvis-K** [comment]
<img width="958" alt="image" src="https://github.com/mail-ecnu/Text-Gym-Agents/assets/18642955/9f5fae30-0d96-45d1-98dd-c211a8f28bd1">

It holds. Do I have to install some specific packages which are missing in `environment.yaml`?

**Comment by Jarvis-K** [comment]
This issue can be solved by 
```
 pip install gym[accept-rom-license]
 AutoROM --accept-license
 ```
 Simplify the README in atari branch

**Comment by ewanlee** [comment]
> This issue can be solved by
> 
> ```
>  pip install gym[accept-rom-license]
>  AutoROM --accept-license
> ```
> 
> Simplify the README in atari branch

Merged into PR #11 with commit [736eff9b1ee5439b558c363aec97bd6d4a900040](https://github.com/mail-ecnu/Text-Gym-Agents/commit/736eff9b1ee5439b558c363aec97bd6d4a900040).

**Comment by Jarvis-K** [comment]
solved
